### DISCUSSION TITLE
Is it possible to support a proxy setting?

### DISCUSSION THREAD

**Discussion by aef5748** [nextcloud/app_api#371] [General]
We deploy the Nextcloud behind the company proxy. 
We already set the proxy environment and updated ca. It can download apps from the apps store via nextcloud. So the network looks workable.
The app_api tried to use the occ command to check or download info.xml at the initial step but failed to connect timeout. 
```
php occ app_api:app:register --force-scopes test-deploy docker_socket_proxy --info-xml https://raw.githubusercontent.com/cloud-py-api/test-deploy/main/appinfo/info.xml --test-deploy-mode --no-ansi --no-warnings
```
Is it possible to add params with proxy to check or download info.xml successfully?
Or is there another way to correct the set proxy configuration?

**Comment by bigcat88** [comment]
I will close this discussion as a duplicate of the same issue: https://github.com/nextcloud/app_api/issues/364

The proposed method should allow to use the proxy setting.
